addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -503,6 +503,26 @@ def _import_invoice_lines(self, invoice, tree, xpath, qty_factor):
             lines_values += self._retrieve_line_charges(invoice, line_values, line_values['tax_ids'])
         return lines_values, logs
 
+    def _retrieve_invoice_line_vals(self, tree, document_type=False, qty_factor=1):
+        # Start and End date (enterprise fields)
+        deferred_values = {}
+        start_date = end_date = None
```

Comment:
Generally better to use `False` to avoid confusion because the ORM will convert to it anyway, and it would lead to the value being written to be different than the value being read afterwards.

Shouldn't be an issue here though